CONTRIBUTING.md

```diff
@@ -179,7 +179,36 @@ virtualenv venv
 # Install openhtf into the virtualenv in dev mode.
 python setup.py develop
 ```
+### MacOS
 
+Install Xcode and Homebrew Mac. Once that is done: 
+
+```bash
+#Install dependent packages
```

Comment:
I would just call these "dependencies". I think that's more clear as OpenHTF is on what could be called the "dependent" side of the relation here.